coordinator/core/marbleapi_test.go

```diff
@@ -222,6 +237,22 @@ func (ms *marbleSpawner) newMarbleAsync(marbleType string, infraName string, sho
 	}()
 }
 
+func (ms *marbleSpawner) verifyCertificateFromEnvironment(envName string, params *rpc.Parameters, opts x509.VerifyOptions) x509.Certificate {
+	p, _ := pem.Decode([]byte(params.Env[envName]))
+	ms.assert.NotNil(p)
+	certificate, err := x509.ParseCertificate(p.Bytes)
+	ms.assert.NotNil(p)
+	ms.assert.NoError(err)
```

Comment:
```suggestion
	ms.require.NotNil(p)
	certificate, err := x509.ParseCertificate(p.Bytes)
	ms.require.NoError(err)
```

Reply:
I left the root CA check & NotBefore check as asserts, is this okay? Not quite sure when to use assert and when to use require...

Reply:
Yep. require stops the test whereas assert just marks it as failed. So you use require if the rest of the test depends on the result. E.g., if p is nil, there would be a nil dereference.